client/actions/events/notifications.js

```diff
@@ -269,23 +269,19 @@ const onRecurringEventCreated = (_e, data) => (
 const onEventUpdated = (_e, data) => (
     (dispatch, getState) => {
         if (data && data.item) {
-            dispatch(eventsUi.scheduleRefetch())
-                .then((events) => {
-                    const selectedEvents = selectors.multiSelect.selectedEvents(getState());
-                    const currentPreviewId = selectors.main.previewId(getState());
-                    const currentEditId = selectors.forms.currentItemId(getState());
-
-                    const loadedFromRefetch = selectedEvents.indexOf(data.item) !== -1 &&
-                        !events.find((event) => event._id === data.item);
-
-                    dispatch(eventsPlanning.ui.scheduleRefetch()).then(() => {
-                        if (!loadedFromRefetch && (currentPreviewId === data.item || currentEditId === data.item)) {
-                            dispatch(eventsApi.fetchById(data.item, {force: true}));
-                        }
-                        dispatch(fetchItemHistoryOnRecurringNotitication(data));
-                    });
-                });
+            const storedEvent = selectors.events.storedEvents(getState())[data.item];
+            const currentEditId = selectors.forms.currentItemId(getState());
+
+            if (get(storedEvent, 'recurrence_id') || get(storedEvent, '_id') !== currentEditId
```

Comment:
It might be better to use `_.get(data, 'recurrence_id')` and `_.get(data, 'item')` rather than from `storedEvent`.

The Event that was updated might not be in the current session's redux store, where as the websocket notification contains the `recurrence_id` if it came from a series